app.js

```diff
@@ -1,23 +1,27 @@
 var userLibrary = [];
-var tempNoteId;
-// this control flow allows us to avoid errors when switching between html pages
+var tempNoteId; // this control flow allows us to avoid errors when switching between html pages
 
-// if a user visits the site on index.html and userIndex and userLibrary both exist in local storage, parse the info and save it to the UserLibrary variable
-if (document.title === "Welcome to Note Fellows!") { //index.html
-  if (localStorage.userIndex  && localStorage.userLibrary) {
+if (document.title === 'Welcome to Note Fellows!') { // index.html
+  if (localStorage.userIndex && localStorage.userLibrary) {
     var userIndex = JSON.parse(localStorage.getItem('userIndex'));
     userLibrary = JSON.parse(localStorage.getItem('userLibrary'));
+    $('#loginForm').append(returnUserForm('loginTemplate', 'returningUserContext'));
+    // $('#loginForm').append(login.returnUserForm('loginTemplate', 'returningUserContext'));
+    // if userIndex and userLibrary exist in localStorage, show returningUserForm
+  }
+  else {
+    // $('#loginForm').append(newUserForm('loginTemplate', 'newUserContext'));
+    $('#loginForm').append(newUserForm());
+    // $('#loginForm').append(login.newUserForm('loginTemplate', 'newUserContext'));
+    // if userIndex and userLibrary DO NOT exist in localStorage, show newUserForm
   }
-  newUserForm(); // why accessing NEW user form if there's already data? shouldn't it be RETURNING user?
 }
-
-// if user visits site on notes.html and userIndex exists, parse it and userLibrary and listen for click on 'noteList' element and run getNote when clicked.
-else if (document.title === "Note Fellows") { // notes.html
+else if (document.title === 'Note Fellows') { // notes.html
   if (localStorage.userIndex) {
     var userIndex = JSON.parse(localStorage.getItem('userIndex'));
     userLibrary = JSON.parse(localStorage.getItem('userLibrary'));
-   }
-  var el = document.getElementById('noteList');
+  }
+  var el = $('#noteList');
```

Comment:
since you used jQuery here you cannot use vanilla javascript on the variable. You would have to add a event listener through jQuery